test/TesterAzureUtils/AzureMembershipTableTests.cs

```diff
@@ -17,22 +19,28 @@ namespace Tester.AzureUtils
     [TestCategory("Membership"), TestCategory("Azure")]
     public class AzureMembershipTableTests : MembershipTableTestsBase, IClassFixture<AzureStorageBasicTests>
```

Comment:
I can't find the base class (`MembershipTableTestsBase`) in this PR, but it should have changed I think, as you added a parameter to its constructor.
Also, I wanted to make sure the logging factory is being correctly disposed.

Reply:
It is odd. But I will make sure the factory is disposing